deploy/operator_dev_cluster.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: member-operator-v0.1
+  namespace: toolchain-member-operator
+spec:
+  lookupPolicy:
+    local: false
+  tags:
+  - from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/codeready-toolchain/member-operator-v0.1:member-operator
+    importPolicy:
+      scheduled: true
+    name: member-operator
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: member-operator
+  name: member-operator
+  namespace: toolchain-member-operator
+spec:
+  replicas: 1
```

Comment:
would it be better to keep `replicas=2`, so that there will be two pod running for member-operator and all our code test against concurrency.  Also, to test sync between member and host .. replica=2 is more meaningful.

Reply:
@nurali-techie, good point, but this is going to work with leader election, this means at any given time, there'll be just one controller which is acting and others are waiting to get elected as master

Reply:
@dipak-pawar as discussed, either keep `replica=2` OR find some rolling update strategy to make sure we have High-Availability during rolling update :+1: